### DISCUSSION TITLE
Is it possible to use nanostores via CDN?

### DISCUSSION THREAD

**Discussion by AllanChain** [ai/nanostores#109] [Q&A] [answered]
I'm trying to use nanostores as an external library, and load it via unpkg or something. But it seems that there is no UMD bundle provided. Is there any plan to provide one (or one for each framework) ?

**Comment by ai** [comment, marked as answer]
Not in the current priority. We need to replace `process.env.NODE_ENV` to some another mechanism and need to create a special release tool for that. It could take a while.
